assets/setup-wizard/features/use-actions-navigator.test.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import { render } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import useActionsNavigator from './use-actions-navigator';
+
+const actionsSample = [
+	{
+		label: 'a',
+		action: () => Promise.resolve(),
+	},
+	{
+		label: 'b',
+		action: () => Promise.resolve(),
+	},
+];
+
+describe( 'useActionsNavigator', () => {
+	const TestComponent = ( { actions } ) => {
+		const { label, percentage } = useActionsNavigator( actions );
+
+		return (
+			<div>
+				<div>{ label }</div>
+				<div>{ percentage }</div>
+			</div>
+		);
+	};
+
+	it( 'should render the first step properly', () => {
```

Comment:
The tests for this hook are very incomplete. The reason is that something is not working properly because the promises and the timeouts. I'd also prefer to use the `@testing-library/react-hooks`, but it was having many problems too.

I tried to update the test dependencies to see if it was fixed, but I had even more problems trying to update it.